src/arch-clap.ts

```diff
@@ -126,18 +131,25 @@ class XarcModuleDev {
   updateFeatures() {
     this.hasEslint = this.appHasDevDeps(...Object.keys(eslintDevDeps));
     this.hasTypeScript = this.appHasDevDeps(...Object.keys(typeScriptDevDeps));
+    this.hasTypedoc = this.appHasDevDeps(...Object.keys(typedocDeps));
   }
 
   setupXclapFile() {
+    const saveFile = (name, content) => {
+      if (!Fs.existsSync(name)) {
```

Comment:
It's unlikely that this would happen, but this is susceptible to a race condition if another process writes the file between the `exists` check and `writeFile` call. A safer approach would be to use the [`ax` or `wx`](https://nodejs.org/docs/latest-v12.x/api/fs.html#fs_file_system_flags) flag with `writeFile` and catch the `EEXIST` error.

Reply:
yeah, the use case here is safe with that scenario though.